### DISCUSSION TITLE
Yet another new alien faction proposal: the Bashkmul

### DISCUSSION THREAD

**Discussion by roadrunner56** [endless-sky/endless-sky#7983] [Ideas]
**EDIT**
I've created the obligatory lore doc for the Bashkmul that contains the most detailed and up-to-date info on the faction. You can take a look here: https://docs.google.com/document/d/1sRW082nXCp0B_TmpIekTWCNlkxlVDhgE1DLDKe1ps80/edit?usp=sharing
 
***DISCLAIMER***

This faction isn’t even close to being PRed or in any sort of playable state. All that exists of them is this post and some messages I made on ESC and in DMs. I’m making this post mostly to test the waters and see what other people think before I start any serious work on these guys.

First off, a big thanks to everyone who helped me develop ideas for this faction. I’d especially like to thank JarlDane, who was instrumental in helping create many of the concepts outlined below. Thanks also to Saugia, Arachi, Zitchas, RisingLeaf, 1010todd, MasterOfGrey and everyone else who commented on the Bashkmul. You guys provided much-needed feedback and advice.

**Intro**

The idea for this faction came up in discussions on ESC about how we had seen several instances of the Drak failing to prevent war and extinction, but had yet to see any situation where the Pug had failed on remotely the same scale. I then thought of a situation parallel to the human civil war: two factions that had fought a massive war against each other, only for the Pug to intervene. One key difference in this scenario is that the factions are different species that originated on different planets in the same region. The major difference, however, is that instead of the Pug mounting an ‘invasion’, they would instead pretend to destroy something much more precious to these aliens. 

**The Concept**

In the region that these aliens occupy, the stars are unusually hot and can be seen during both day and night on most planets. This led to a veneration of stars and their movement across the sky in both of the factions’ cultures. The Pug, wanting to manipulate this, pretended that they were going to destroy the stars and use them as fuel in order to encourage the two aliens to unite against them. To convince them of this fact, they ‘sped up’ the life-cycles of these stars, making it appear that they were dying. The two species united against the Pug and formed the Bashkmul, but once the ‘invasion’ was defeated and their mission complete, the Pug found that they were unable to reverse the death of these stars. 
The Pug had come to this galaxy countless millennia ago, but even this long time span left them unaware of still older things in the Milky Way. One of these was an ancient starfaring race known as the Rulei. They had mastered the use of stars for their own purposes, engineering stars in Bashkmul space to the point that normal physical rules as the Pug understood them did not apply. The Pug, having manipulated galactic phenomena before in other regions of the universe, assumed that this time would be no different. But they did not understand that these stars had been engineered to the point that they no longer operated by the Pug’s rules, and so their game doomed all life in this part of the galaxy. 
After hastening the death of the stars in the region, and finding themselves unable to fix the problem, they shuffled a few of the two species to another world outside of the region. They haven’t bothered to try to save the rest, in the same way a child who’s broken a toy gets bored with it and doesn’t want to play with it anymore. The player finds the two species desperately trying to eke out a living on their increasingly cold and dying worlds, knowing that their civilizations are heading towards inevitable collapse.
These aliens would be found south of Gegno space, in territory that the Rulei were once present in. The individuals that the Pug moved to prevent extinction would live on a planet in an area to the west of Bashkmul territory. This entire area was untouched by the Rulei, but is only accessible by jump drive from Bashkmul space. This area is also connected to the edges of inhabited Gegno space. 

**The Aliens**

The aliens themselves are the Aert and the Rigra, two species that discovered spaceflight within a thousand years of each other. 
The Aert are large, 6-legged creatures, about two meters long. Their arms and hands have large, slothlike paws made for digging, reflecting their underground origin. They have a wide tail and short brown fur, along with a mole-like nose. Their eyesight is poor, but they have a very strong sense of smell and touch as a result of a lack of light in their original environment. 
Their society is formed of groups of about 100-200, which are part of wider districts with populations in the millions. They still make their homes underground, but live aboveground for all other purposes. They have a sort of ‘don’t rock the boat’ attitude, and don’t tend to be very adventurous. As such their development of spaceflight was slower than other species of comparable age. They were the second of the two to discover spaceflight. In an effort to kickstart the Aert’s technological progression, the Pug linked Aert space with the Rigra via a wormhole, with the hope that the Rigra’s more aggressive pursuit of progress may influence the Aert.
Before the war, they had a planet with vast agricultural resources, which the Rigra attacked after a plague on their own worlds wiped out their food supply. However, this planet was also essential for feeding millions of Aert, leading to the normally peaceful Aert going to war. 
They have a fatalistic attitude towards their current situation. There has been a religious revival of sorts as well, with many Aert reverting back to previous spiritual traditions.
The Rigra are about a meter tall, with rough, greenish-brown skin. They walk on their hind legs, but can easily switch to all fours if needed. Their origins are in the jungle, but are adaptable to several different environments. Two large eyes on their face and two smaller ones on the back of their heads allow them to maintain a 360 degree view of their surroundings. This is due to their evolutionary history as prey to larger species. 
This combined with their small size and weak physical strength has led to a culture of doing what it takes to survive. Their culture also places a lot of importance on technology and discovery, as this is what allowed the Rigra to rise above their precarious position in the food chain. 
After meeting the Aert, there was some debate as to how to respond. Some wanted to attack them for their resources and planets, with their reasoning being that if the Aert were allowed to grow in power, they may pose a threat to the Rigra in the future; better to stamp them out now while they were weak. Fortunately, the Rigra soon realized that the Aert were unlikely to ever attempt any sort of attack on them, and saw greater value in learning more about this strange new species.
Some Rigra still maintain hope that they would be able to restore their stars to their original state, but all efforts have currently been futile. 
Both share a reverence for stars and astronomy as part of their traditional cultures. Strikingly, many historical civilizations, both Aert and Rigra, had creation myths involving a group of god-like beings forming the stars. Some now hold that these beings will come back, restore the stars, and punish the Pug for their crimes.

**Relationships**

Pug – For rather obvious reasons the Pug and the Bashkmul are not on good terms. If the Pug are asked about the Bashkmul, they flip-flop between a blasé sort of indifference and a deep insecurity about their failure.

Quarg – The Quarg have unfinished ringworlds in Rigra and Aert space. Both are currently only skeletal structures, and as such are not able to house more than a fraction of the Bashkmul's population. After the war the Quarg have attempted to speed up the construction of the rings in order to provide a refuge for the Bashkmul. They have a deep disgust of the Pug’s treatment of the Bashkmul and this is reflected in any conversations you may have with them. However, they do not intervene due to their fear of provoking the Pug. As a result, the Bashkmul are distrustful of the Quarg, seeing them as cowardly for not directly confronting the Pug.

Player – The player is initially attacked by the Bashkmul both for their jump drive and due to a distrust of outsiders as a result of the Pug intervention. Perhaps jettisoning cargo could increase reputation, but I’m not sure if this is even possible as a mechanic.

**Timeline**

1400 – Rigra launch first interstellar colonization efforts
1500 – Rigra discover hyperdrive
1600 – Rigra establish first successful colonies in surrounding stars, ringworld in Rigra space begins construction.
1700s – 2000s – Several expansion attempts are made further into Rigra space, many unsuccessful. A few, however, manage to establish themselves.
2500 – Aert discover hyperdrive. The Quarg begin to construct a ringworld in Aert space. 
2600 – Aert begin interstellar colonization. Pug form wormhole between the Aert and Rigra.
2650 – Rigra discover the wormhole and make first contact with the Aert. A peace treaty is signed,  outlining the boundaries of possible expansion for either species.
2700 – Rigra are at the height of their interstellar empire, conducting trade with the now fast-growing Aert. 
2750 – Rigra are hit by an agricultural plague that spreads throughout their planets, devastating their agricultural supply. In an attempt to prevent mass starvation, they attack an Aert agricultural planet, starting the Aert-Rigra war.
2770 – The Pug step in and accientally cause the slow death of all of the stars in the region, dooming the Aert and Rigra. The two unite against the Pug, calling themselves the Bashkmul. 

**Possible storyline**

This is probably the most nebulous part of the proposal, and the one most subject to change.

The current plan for the Bashkmul storyline depends on the Xapelaux. Grey has mentioned a possible campaign option where the Xapelaux rediscover jump drive technology. For the Bashkmul storyline, I’m planning to have the Xapelaux produce jump drives again in order to ferry them out of their dying region and into the new, untouched western region. This would help the Xapelaux overcome their fear of leaving the Brilliance, and also give them the opportunity to make friends with species from outside of the Brilliance. 

The new region that they enter is connected to Gegno space. Saugia and I are planning for some small Bashkmul-Gegno interactions after the Bashkmul migrate there.

**Misc. Notes**

Besides maybe the Korath, there aren’t really any species at real risk of extinction in the current game. I wanted to explore this concept further.

Currently the Pug are so powerful to the point that no one wants to touch them with a 10 foot pole, and understandably so. Having a faction of 4D chess masters that never fail isn’t conductive to good storytelling, and is more importantly boring. I wanted to show the Pug making mistakes in a way that was both plausible and in-character for them. 

I’m thinking that the Bashkmul would be around tier 1.2, but I’m not dead set on any level. Balancing and stats aren’t really my wheelhouse. 

The name of the Bashkmul comes from the Albanian word for ‘together’, _bashk_. Hopefully no one playing ES speaks Albanian and notices the really obvious name. All the other names were just made up because they sounded cool.

I’m very open to criticism and comments. The main reason I’m posting this is to get feedback on my ideas and make sure I didn’t miss anything obvious.

**Edit**
I've made a very WIP repository for the Bashkmul, available here: https://github.com/roadrunner56/Bashkmul/tree/main

**Comment by Luuiscool4567** [comment]
Not going to lie, having the Pug "accidentally" screwing up that badly doesn't feel right. These guys can litterally make wormholes given a couple of weeks without nuking the local hyperspace systems. Any species past tier 3 would probably be smart enough to fix the problem if they caused bit in the first place. There's also the idea that the Pug are just lying.

**Reply by Zitchas** [reply]
Higher tech than everyone else doesn't actually mean they have the capability to do everything; or that they are brilliant, or anything else. In this case, it just means they've been around longer than almost everyone else. They are implied to have a fairly impressive amount of ressources somewhere, but nothing says that has to be infinite. In fact, the fact that they are reluctant to bring it to bear (for instance, the fairly weak resonse to the player pirating their Arfectas) could be taken to imply that they have rather few resources to use for direct application, and that they rely on manipulation and gamesmanship because they can't actually field an overwhelming navy.

Also, being able to build wormholes doesn't necessarily mean being able to precisely tweak the inner workings of a star. They're vastly different things, and while both being extremely advanced tech, they could be in completely different technological paradigms for all we know. Creating wormholes could be something of a one-trick endeavor: Very powerful, but limited.

Beyond that, having the Pug actually make a mistake would be a welcome addition to the lore. As they stand now, the Pug are boring, one-dimensional dues-ex-mcahina forces. Right now, the Pug are Perfect, the Quarg are powerful but overbearing and timid, and the Drak are powerful but just keep failing again and again and again and so they're rather mopey and depressed. Unless we want to have a perfect faction, the Pug need some failures.

**Reply by roadrunner56** [reply]
...what Zitchas said.

**Reply by mOctave** [reply]
Perhaps this event of the Pug messing up so badly is even one of the events that has _led_ to them being very cautious in other places and not committing all of their resources to anything (not necessarily a sound argument, but possible).

**Reply by Ferociousfeind** [reply]
The Pug messing up _does_ rub me the wrong way- "what do you mean you fucked up the star? what did you expect?" I'd like more exposition on the vaguely-alluded-to aliens that had tampered with the stars before, maybe a bit of technobabble as to why the pug further meddling irreversibly broke the stars.

Maybe if stars are established as "difficult to accidentally break", I'd be less inclined to assume they are like porcelain or fine china- something you obviously don't touch because you're obviously going to break it, and more like, say, rubber- something you can throw around and make bounce, if you know its limits, but if it's tampered with beforehand (say, cooled) it'll shatter much more easily than expected.

**Reply by Saugia** [reply]
I wanted to keep it secret for now, but to help shed a little light. Our lore tie-in is that the Rulei are said ancient race, and one of the aspects of their lore is they consumed everything they could or had the means to. Now, casually consuming stars would be at the far end of their "we tried this" spectrum, though the difficulty wouldn't stop them from attempting it. 

Think of it this way: if the Korath were able to supernovae stars, I wouldn't put it past the Rulei, who are higher tier, to be able to consume parts of them in some way. So the idea was the Bash will be located south of Gegno (Rulei) territory, and the Rulei had at some points attempted to, or partially successfully, consume matter from their stars.

(I'd prefer if this lore didn't get out too much from this PR as Rulei lore is still somewhat in its pre-reviewed stage, but all the generalities of it I'm planning on existing.)

**Reply by Zitchas** [reply]
Was this pre-planned, or developing? Based on my initial readings here, the Rulei struck me as being a potential area cross-tie between these two areas, which would be neat.

**Reply by Saugia** [reply]
Developing. It wasn't pre-planned, but the Rulei fit the bill for the ancient race here. Works out so we don't have to add yet another "ancient dead race" to our growing list of them. I suppose you could say the Rulei are the Sheragi of the "east".

**Comment by RisingLeaf** [comment]
I really like the story and the concept of the pug not being perfect, but this line is bugging me, I do not think that the pug would just go away after they did something so terrible.
> They haven’t bothered to try to save the rest, in the same way a child who’s broken a toy gets bored with it and doesn’t want to play with it anymore.

Another thing I want to ask is, why they Quarg would not ask the Archons for help?

**Reply by roadrunner56** [reply]
The Pug have a different conception of morality than most other species. Their identity is communal, meaning that the survival of the species is all that matters. The death of individuals is viewed as wasteful, but not necessarily immoral. As long as there's a viable breeding population, they don't really care. This is why the Pug are totally cool with letting the Wanderers fend for themselves with the Unfettered and Automata, as those aren't existential-level threats to the Wanderers, even though their attacks caused massive casualties. It's also why they're fine with throwing away their lives at the end of FW, as again, individuals aren't important; the goals of the entire species are.
The Archons don't get involved for two reasons; one, they don't want to trigger a war with the Pug, as they somewhat view the Bashkmul as the Pug's turf, and two, the only way they could plausibly help is to create a hyperlink to the western region, which is currently empty. However, it is connected to Gegno space, and the Drak want to prevent the Gegno from coming into contact with the Bashkmul, as they don't want to potentially destabilize the Gegno's civil war further. 
Both sides thus content themselves with saving a few Bashkmul in the Quarg rings, and don't attempt to do more.

**Comment by riyan-gendut** [comment]
I feel that for the Pug to reasonably not know that the law of physics in the region is fucked up the event can't really be recent, it should occur much earlier in the Pug's entry to the galaxy. I guess it says something about their immense arrogance to not even test for messed up physics, but realistically to hang the threat of stellar death on the Bashkmul there's no need to destabilize the entire region, just "killing" a couple stars would've suffice to prove that they're real threat.

(is this considered necro?)

**Reply by roadrunner56** [reply]
You're right - I have an updated lore doc where the Pug only mess around with the 4 stars in Aert space. What caused the Aert's stars to be different happened before the Pug came to the Milky Way - in short, an ancient species called the Rulei messed around with them millions of years ago, and this tampering was unknown to the Pug.
This discussion is extremely out of date lol, I'd recommend looking at the lore doc linked above for the most recent info on the Bashkmul.
